editoast/src/core/simulation.rs

```diff
@@ -113,6 +137,84 @@ impl PhysicsRollingStock {
     }
 }
 
+fn compute_rolling_resistance(
+    traction_engine: &RollingStock,
+    towed_rolling_stock: &Option<TowedRollingStock>,
+    total_mass: Option<f64>,
+) -> RollingResistance {
+    if let (Some(towed_rolling_stock), Some(total_mass)) = (towed_rolling_stock, total_mass) {
+        let traction_engine_rr = &traction_engine.rolling_resistance;
+        let towed_rs_rr = &towed_rolling_stock.rolling_resistance;
+        let traction_engine_mass = traction_engine.mass; // kg
+
+        let mass_carriage = total_mass - traction_engine_mass; // kg
+
+        let rav_a_te = traction_engine_rr.A * 1000.0; // N
+        let rav_b_te = traction_engine_rr.B * 1000.0 * 3.6; // N/(m/s)
+        let rav_c_te = traction_engine_rr.C * 1000.0 * 3.6 * 3.6; // N/(m/s)²
```

Comment:
```suggestion
        let rav_a_te = traction_engine_rr.A * 1000.0; // convert from kN to N
        let rav_b_te = traction_engine_rr.B * 1000.0 * 3.6; // convert from kN/(km/h) to N/(m/s)
        let rav_c_te = traction_engine_rr.C * 1000.0 * 3.6 * 3.6; // convert from kN/(km/h)² to N/(m/s)²
```

I suggest specifying the convertion, not only the final unit.